src/openforms/authentication/contrib/yivi_oidc/tests/test_admin.py

```diff
@@ -0,0 +1,268 @@
+
+            # Expect that the export was successful
+            self.assertEqual(export_response.status_code, 200)
+
+        with self.subTest("Remove local attributegroups"):
+            # Remove local attributegroups, so we can add them via import
+            attributeGroup1.delete()
+            attributeGroup2.delete()
+            self.assertEqual(len(AttributeGroup.objects.all()), 0)
+
+        with self.subTest("Import attributegroups"):
+            # Go to import page
+            import_page = self.app.get(self.admin_import_url, user=user)
+            import_form = import_page.forms[1]
+
+            # Upload the data file, set the file type, and import
+            import_form["import_file"] = (
+                "testfile.json",
+                export_response.body,
+                "application/json",
+            )
+            import_form["input_format"] = self._get_value_for_select_field(
+                import_form["input_format"], "json"
+            )
+            import_response = import_form.submit()
+            self.assertEqual(import_response.status_code, 200)
+
+        with self.subTest("Confirm import of attributegroups"):
+            # After submitting the import form, we are send to a confirmation form.
+            # This confirmation form includes a preview of the data being imported.
+            import_preview = import_response.pyquery(".import-preview")
+            self.assertEqual(len(import_preview), 1)
+
+            # The `.import-preview` contains both attribute groups that we are importing.
+            import_preview_rows = import_response.pyquery(".import-preview tr.new")
+            self.assertEqual(len(import_preview_rows), 2)
+
+            # Confirm the import preview, and perform the actual import.
+            confirm_form = import_response.forms[1]
+            confirm_response = confirm_form.submit()
+            self.assertEqual(confirm_response.status_code, 302)
+
+        with self.subTest("Check imported data"):
+            # Expect that 2 attribute groups are imported
+            self.assertEqual(len(AttributeGroup.objects.all()), 2)
+
+            # Expect that the imported attribute groups have the same content as the
+            # exported attribute groups
+            group1 = AttributeGroup.objects.get(
+                uuid="12345678-abcd-1234-abcd-1234567890ab"
+            )
+            self.assertEqual(group1.name, "group1")
+            self.assertEqual(group1.description, "For retrieving group 1 data")
+            self.assertEqual(group1.attributes, ["foo", "bar", "foo.bar"])
+
+            group2 = AttributeGroup.objects.get(
+                uuid="12345678-abcd-1234-abcd-1234567890cd"
+            )
+            self.assertEqual(group2.name, "group2")
+            self.assertEqual(group2.description, "For retrieving group 2 data")
+            self.assertEqual(group2.attributes, ["zoo", "zar"])
+
+    def test_successful_update_attribute_group_using_import(self):
```

Comment:
unless there's a lot of custom code going on in the import-export code, these tests aren't really necessary since they're behaviour should be covered by the django-import-export testsuite itself.